docs/application/native/guides/ui/dali/videoview.md

```diff
@@ -108,6 +110,19 @@ newMap.Insert( "volumeRight", 0.5f );
 mView.SetProperty( VideoView::Property::VOLUME, newMap );
 ```
 
+To change the display mode, use the `PLAYER_DISPLAY_MODE` enums (in [mobile](../../../api/mobile/latest/group__CAPI__MEDIA__PLAYER__MODULE.html) and [wearable](../../../api/wearable/latest/group__CAPI__MEDIA__PLAYER__MODULE.html) applications) .
```

Comment:
One additional point to check : The link was to "group__CAPI__MEDIA__PLAYER__**DISPLAY**__MODULE.html" at first, and now it is heading to "group__CAPI__MEDIA__PLAYER__MODULE.html". Is `PLAYER_DISPLAY_MODE` really linked to "group__CAPI__MEDIA__PLAYER__MODULE.html", not  "group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html"?